docs/quick-starts/setup-quickstart.md

```diff
@@ -5,8 +5,43 @@ sidebar_position: 1
 
 # Initial setup
 
-Steps
+This page walks through the initial setup of your Eppo instance. In this guide, you'll learn how to access Eppo, connect your data warehouse, create your first entity, and connect historical experiment assignment logs.
```

Comment:
I wonder whether we should combine this with the "Create first metric" page. Currently there isn't all that much content and also it seems like the steps here aren't really needed to start using Eppo for feature flagging.